Error while installing

<img width="766" alt="Jepretan Layar 2021-10-07 pukul 16 14 46" src="https://user-images.githubusercontent.com/49934033/136346162-6ea22a02-7013-47d5-8f64-4aa6dbdad278.png">
hello i found the error while installing using homebrew

`Last 15 lines from /Users/gusadi/Library/Logs/Homebrew/coherent-swift@0.5.10/01.make:
                            ~~~~~ ^~~~~
/private/tmp/coherent-swiftA0.5.10-20211009-29260-1efg7g7/.build/checkouts/swift-syntax/Sources/SwiftSyntax/RawSyntax.swift:157:55: error: value of type 'CSyntaxNode' (aka 'swiftparse_syntax_node_t') has no member 'range'
      let end = utf8.index(begin, offsetBy: Int(cnode.range.length))
                                                ~~~~~ ^~~~~
/private/tmp/coherent-swiftA0.5.10-20211009-29260-1efg7g7/.build/checkouts/swift-syntax/Sources/SwiftSyntax/RawSyntax.swift:873:32: error: value of type 'CSyntaxNode' (aka 'swiftparse_syntax_node_t') has no member 'range'
    let byteLength = Int(cnode.range.length)
                         ~~~~~ ^~~~~
/private/tmp/coherent-swiftA0.5.10-20211009-29260-1efg7g7/.build/checkouts/swift-syntax/Sources/SwiftSyntax/SyntaxParser.swift:195:57: error: missing argument for parameter #3 in call
    let c_top = swiftparse_parse_string(c_parser, source)
                                                        ^
                                                        , <#Int#>
_InternalSwiftSyntaxParser.swiftparse_parse_string:1:13: note: 'swiftparse_parse_string' declared here
public func swiftparse_parse_string(_: swiftparse_parser_t!, _ source: UnsafePointer<CChar>!, _ len: Int) -> swiftparse_client_node_t!
            ^
make: *** [coherent-swift] Error 1

`

--- a/Package.swift
+++ b/Package.swift
@@ -26,7 +26,7 @@ let package = Package(
         .package(
 			name: "SwiftSyntax",
             url: "https://github.com/apple/swift-syntax.git",
-            .exact("0.50400.0")
+            .exact("0.50500.0")
         )
     ],
     targets: [

